manufacturing/enclosed_build/enclosure/README.md

```diff
@@ -17,11 +41,7 @@ The internal equipment bracket is folded from a single sheet. The bracket is des
 Acrylic sheets are either fastened to or hung from the bracket to allow for design adjustments while utilizing the same
 bracket.
 
-**#TODO: present CAD information properly, in a structured, attractive way**
-
-The part files are available in the CAD folder. [.ipt for enclosure](CAD/ventilator_enclosure.ipt),
-[.ipt for equipment bracket](CAD/equipment_bracket.ipt), [.ipt for closure panel](CAD/ventilator_closure_panel.ipt).
-The assembly file is also available [.iam for enclosure assembly](CAD/enclosure_assembly.iam).
+We are currently on v0.3. However, the design is being altered. Changes and updates are being tracked in the issue tickets [#957](https://github.com/RespiraWorks/Ventilator/issues/957) (this is an umbrella ticket and contains references to some of the other tickets), [#958](https://github.com/RespiraWorks/Ventilator/issues/958), [#959](https://github.com/RespiraWorks/Ventilator/issues/959), [#961](https://github.com/RespiraWorks/Ventilator/issues/961), [#962](https://github.com/RespiraWorks/Ventilator/issues/962), [#963](https://github.com/RespiraWorks/Ventilator/issues/963), [#964](https://github.com/RespiraWorks/Ventilator/issues/964), [#965](https://github.com/RespiraWorks/Ventilator/issues/965), [#970](https://github.com/RespiraWorks/Ventilator/issues/970).
```

Comment:
I don't think we need to link to tickets from here. Tickets are ephemeral and this will be out of date very fast.